npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -46,9 +46,12 @@ def self.npm_version_numeric(lockfile)
         npm_version_numeric_npm6_or_higher(lockfile)
       end
 
-      sig { params(lockfile: DependencyFile).returns(Integer) }
+      sig { params(lockfile: T.nilable(DependencyFile)).returns(Integer) }
       def self.npm_version_numeric_npm6_or_higher(lockfile)
-        lockfile_content = T.must(lockfile.content)
```

Comment:
If the lockfile doesn't have content instead of Sorbet check which causing error we want to return our default version.